docs/Makefile

```diff
@@ -1,50 +1,39 @@
-# Minimal makefile for Sphinx documentation
-# Carefully tested on 18 May 2017
+# Makefile for Sphinx documentation
+#
 
-# You can set these variables from the command line.
-SPHINXOPTS    =
-SPHINXBUILD   = sphinx-build
-SPHINXPROJ    = broadbean
-SOURCEDIR     = source
-BUILDDIR      = build
-GH_PAGES_SOURCES = docs broadbean
+# You can set these variables from the command line, and also
+# from the environment for the first two.
+SPHINXOPTS    ?=
+SPHINXBUILD   ?= sphinx-build
+SOURCEDIR     = .
+BUILDDIR      = _build
+
+# this is to to autmagic gh-pages
```

Comment:
Double to

Reply:
Actually that variable is never used now as far as I can see

Reply:
okay thanks for checking, I'll remove